optd-persistent/src/bin/migrate_test.rs

```diff
@@ -0,0 +1,23 @@
+use optd_persistent::{migrate, TEST_DATABASE_URL};
+use sea_orm::*;
+use sea_orm_migration::prelude::*;
+
+#[tokio::main]
+async fn main() {
+    let _ = std::fs::remove_file(TEST_DATABASE_URL);
+
+    let db = Database::connect(TEST_DATABASE_URL)
+        .await
+        .expect("Unable to connect to the database");
+
+    migrate(&db)
+        .await
+        .expect("Something went wrong during migration");
+
+    db.execute(sea_orm::Statement::from_string(
+        sea_orm::DatabaseBackend::Sqlite,
+        "PRAGMA foreign_keys = ON;".to_owned(),
+    ))
+    .await
+    .expect("Unable to enable foreign keys");
```

Comment:
Is there a reason we need a second database file?

Reply:
It is for test specifically. I think we need a second db file only for testing. But this file is kinda out of date, will update it.

Reply:
Ah I see I was sort of just assuming we would use 1 file for everything including tests, this works too.